plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/express-payment/test/checkout-express-payment.tsx

```
@@ -295,70 +284,93 @@ describe( 'CheckoutExpressPayment', () => {
-						stripe: { name: 'stripe' },
-					},
-				} );
+		it( 'should render 1 skeleton buttons when calculating a partial update', () => {
+			mockUseSelect.mockReturnValueOnce( {
+				isCalculating: true,
+				isProcessing: false,
+				isAfterProcessing: false,
+				isBeforeProcessing: false,
+				isComplete: false,
+				hasError: false,
+				availableExpressPaymentMethods: {},
```

Biome (1.9.4)

[error] This property value named availableExpressPaymentMethods is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named availableExpressPaymentMethods


(lint/suspicious/noDuplicateObjectKeys)